ddl/db_change_test.go

```diff
@@ -585,10 +585,22 @@ func (s *testStateChangeSuite) testControlParallelExecSQL(c *C, sql1, sql2 strin
 		for {
 			kv.RunInNewTxn(s.store, false, func(txn kv.Transaction) error {
 				m := meta.NewMeta(txn)
+				addIdxLen := int64(0)
+				if addIdxJobCnt != 0 {
```

Comment:
addIdxJobCnt is used as a bool value. Why use int type?

Reply:
If we only need an `add index` operation, then `addIdxJobCnt` is 1.

Reply:
So how about using a bool value named needAddIndexOpt?